nextflow.config

```diff
@@ -1,197 +1,253 @@
+        podman.enabled         = false
+        shifter.enabled        = false
+        charliecloud.enabled   = false
+    }
+    podman {
+        podman.enabled         = true
+        docker.enabled         = false
+        singularity.enabled    = false
+        shifter.enabled        = false
+        charliecloud.enabled   = false
+    }
+    shifter {
+        shifter.enabled        = true
+        docker.enabled         = false
+        singularity.enabled    = false
+        podman.enabled         = false
+        charliecloud.enabled   = false
+    }
+    charliecloud {
+        charliecloud.enabled   = true
+        docker.enabled         = false
+        singularity.enabled    = false
+        podman.enabled         = false
+        shifter.enabled        = false
+    }
+    test            { includeConfig 'conf/test.config'      }
+    test_full       { includeConfig 'conf/test_full.config' }
+    test_full_build { includeConfig 'conf/test_full_build.config' }
+
 }
 
-// Export this variable to prevent local Python libraries from conflicting with those in the container
+// Export these variables to prevent local Python/R libraries from conflicting with those in the container
+// The JULIA depot path has been adjusted to a fixed path `/usr/local/share/julia` that needs to be used for packages in the container.
+// See https://apeltzer.github.io/post/03-julia-lang-nextflow/ for details on that. Once we have a common agreement on where to keep Julia packages, this is adjustable.
+
 env {
-  PYTHONNOUSERSITE = 1
+    PYTHONNOUSERSITE = 1
+    R_PROFILE_USER   = "/.Rprofile"
+    R_ENVIRON_USER   = "/.Renviron"
+    JULIA_DEPOT_PATH = "/usr/local/share/julia"
 }
 
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
+def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
-  enabled = true
-  file = "${params.tracedir}/execution_timeline.html"
+    enabled = true
+    file    = "${params.tracedir}/execution_timeline_${trace_timestamp}.html"
 }
 report {
-  enabled = true
-  file = "${params.tracedir}/execution_report.html"
+    enabled = true
+    file    = "${params.tracedir}/execution_report_${trace_timestamp}.html"
 }
 trace {
-  enabled = true
-  file = "${params.tracedir}/execution_trace.txt"
+    enabled = true
+    file    = "${params.tracedir}/execution_trace_${trace_timestamp}.txt"
 }
 dag {
-  enabled = true
-  file = "${params.tracedir}/pipeline_dag.svg"
+    enabled = true
+    file    = "${params.tracedir}/pipeline_dag_${trace_timestamp}.html"
 }
 
 manifest {
-  name = 'nf-core/rnafusion'
-  author = 'Martin Proks'
-  homePage = 'https://github.com/nf-core/rnafusion'
-  description = 'Nextflow rnafusion analysis pipeline, part of the nf-core community.'
-  mainScript = 'main.nf'
-  nextflowVersion = '>=19.10.0'
-  version = '1.2.0'
+    name            = 'nf-core/rnafusion'
+    author          = 'Martin Proks, @praveenraj2018'
```

Comment:
Looking at who contributed to the pipeline tbh, the handle of Praveen should not be here at all. 

Martin / Maxime / Rickard / Phil and mostly you @rannick shuold be here.